eliot-service/eliot/libsymbolic.py

```diff
@@ -103,13 +101,12 @@ def convert_debug_id(debug_id):
         raise BadDebugIDError("invalid_identifier")
 
 
-def parse_sym_file(debug_filename, debug_id, data, tmpdir):
```

Comment:
When Eliot downloads a sym file from symbols.mozilla.org, it has the file in memory and now parses it from memory. Previously, it would have it in memory, save it to disk (which caused other problems), parse it from disk, and then delete the file from disk. But, it's doing this while the file is still in memory, so there's no memory savings to saving it to disk and saving it and loading it from disk probably slowed things down (I never measured this--it would have been really hard to get good measurements).

Anyhow, this removes the whole save it to disk / parse it from disk / remove the temporary file thing.